tsdb/index.go

```diff
@@ -968,6 +988,46 @@ func (itr *measurementSliceIterator) Next() (name []byte, err error) {
 	return name, nil
 }
 
+func (itr *measurementSliceIterator) ToSlice() [][]byte {
+	return itr.names
```

Comment:
Having both `ToSlice` and `Next` on the same object seems prone to accident...

Reply:
Maybe if it was named `RemainingAsSlice` or something that suggests it is part of the iteration interface

Reply:
UnderlyingSlice() is the new name

Reply:
Hmm, UnderlyingSlice still hints to me that the slice is the whole thing instead of eaten away at by `Next`, but ok.